device/nokia/x86_64-nokia_ixr7250e_36x400g-r0/Nokia-IXR7250E-36x100G/0/port_config.ini

```diff
@@ -0,0 +1,21 @@
+# name         lanes                    alias            index  asic_port_name role  speed    coreid  coreportid  numvoq
+Ethernet0      72,73,74,75              Ethernet0        1      Eth0-ASIC0     Ext   100000   1       1           8
+Ethernet1      80,81,82,83              Ethernet1        2      Eth1-ASIC0     Ext   100000   1       2           8
+Ethernet2      88,89,90,91              Ethernet2        3      Eth2-ASIC0     Ext   100000   1       3           8
```

Comment:
Can we change the naming so that the interface names and aliases are in sync with SONiC naming.

Ethernet0      72,73,74,75              Ethernet1/1        1      Eth0-ASIC0     Ext   100000   1       1           8
Ethernet4      80,81,82,83              Ethernet2/1        2      Eth1-ASIC0     Ext   100000   1       2           8
Ethernet8      88,89,90,91              Ethernet3/1        3      Eth2-ASIC0     Ext   100000   1       3           8

Reply:
Based on the conversation on the weekly team meeting, we will address this on a separated PR